### PR TITLE
[OTel] enable instrumentation tests

### PR DESCRIPTION
**Pull request by pvital**
Add logic to enable the instrumentation tests as their migration is finalized. 

In addition, formated the `__init__.py` file and fixed lint violations, using `ruff` (vscode) to:
- Black-compatible code formatting.
- fix all auto-fixable violations, like unused imports.
- isort-compatible import sorting.

### CODE REVIEW

**Comment by pvital** [comment]
This PR depends on PR #570.

File: src/instana/__init__.py [hunk 1]
```diff
@@ -1,38 +1,36 @@
...
+    is_webhook_instrumented,
+)
+from instana.version import VERSION
+
+__author__ = "Instana Inc."
+__copyright__ = "Copyright 2020 Instana Inc."
+__credits__ = ["Pavlo Baron", "Peter Giacomo Lombardo", "Andrey Slotin"]
+__license__ = "MIT"
+__maintainer__ = "Peter Giacomo Lombardo"
+__email__ = "peter.lombardo@instana.com"
```
Comment (GSVarsha) [inline src/instana/__init__.py:28]: I think we should think about changing all the entries except `__license__` 😅

File: src/instana/__init__.py [hunk 2]
```diff
@@ -115,81 +137,91 @@ def lambda_handler(event, context):
...
         # Now get the function and execute it
         if hasattr(handler_module, function_name):
             handler_function = getattr(handler_module, function_name)
             return handler_function(event, context)
         else:
-            print("Couldn't determine and locate default function handler: %s.%s" % (module_name, function_name))
+            print(
+                "Couldn't determine and locate default function handler: %s.%s"
+                % (module_name, function_name)
+            )
```
Comment (GSVarsha) [inline src/instana/__init__.py:153]: Can we use f-strings?

File: src/instana/__init__.py [hunk 3]
```diff
@@ -115,81 +137,91 @@ def lambda_handler(event, context):
...
+        # from instana.instrumentation import pymysql  # noqa: F401
+        # from instana.instrumentation import psycopg2  # noqa: F401
+        # from instana.instrumentation import redis  # noqa: F401
+        # from instana.instrumentation import sqlalchemy  # noqa: F401
+        from instana.instrumentation import (
+            fastapi_inst,  # noqa: F401
+            flask,  # noqa: F401
+            logging,  # noqa: F401
+            urllib3,  # noqa: F401
+        )
```
Comment (GSVarsha) [inline src/instana/__init__.py:196]:
```suggestion
        from instana.instrumentation import (
            flask,  # noqa: F401
            logging,  # noqa: F401
            urllib3,  # noqa: F401
        )
```

File: src/instana/__init__.py [hunk 4]
```diff
@@ -115,81 +137,91 @@ def lambda_handler(event, context):
...
+        and len(sys.argv) > 0
+        and (os.path.basename(sys.argv[0]) in do_not_load_list)
+    ):
         if "INSTANA_DEBUG" in os.environ:
-            print("Instana: No use in monitoring this process type (%s).  "
-                  "Will go sit in a corner quietly." % os.path.basename(sys.argv[0]))
+            print(
+                "Instana: No use in monitoring this process type (%s).  "
+                "Will go sit in a corner quietly." % os.path.basename(sys.argv[0])
+            )
```
Comment (GSVarsha) [inline src/instana/__init__.py:216]:
```suggestion
            print(
                f"Instana: No use in monitoring this process type ({os.path.basename(sys.argv[0])}. Will go sit in a corner quietly."
            )```

File: src/instana/__init__.py (hunk 1 above)
Comment (pvital) [reply src/instana/__init__.py:28]: We should also do that, but first, we need to investigate internally how to proceed in this case. 
For now, I'd like to keep it as is.

**Review by GSVarsha — APPROVED** [review]
Looks good to me!

**Review by CagriYonca — APPROVED** [review]

**Pull request merged**